lib/util/urltils.js

```diff
@@ -162,6 +162,23 @@ module.exports = {
     }
   },
 
+  /**
+   * Obfuscates path parameters with regex from config
+   *
+   * @param {Config} config The configuration containing the regex
+   * @param {string} requestPath The Path to be obfuscated
+   * @returns {string} The obfuscated path
+   */
+  obfuscatePath: function obfuscatePath(config, requestPath) {
+    if (typeof requestPath === 'string') {
+      const regex = config.url_obfuscation_regex
+      if (regex) {
+        return (requestPath = requestPath.replace(regex, '/***'))
```

Comment:
this feels very limiting.  I picture the user case of such:

`/api/v1/users/sensitive/edit`

there's currently no way for me to apply the url obfuscation regex to result in `/api/v1/users/**/edit`.  I'm leaning towards that configuration stanza being more open like this

```
url_obfuscation: {
  regex: '(/api/v1/users/)(.*)(/edit)',
  replacement: '$1**$3'
}
```